wd_demo.au3

```diff
@@ -343,6 +343,10 @@ Func DemoElements()
 
 	; Check if first DIV element is visible, as it can hide all sub elements in case when COOKIE aproval message is visible
 	_WD_WaitElement($sSession, $_WD_LOCATOR_ByXPath, '//body/div[1][@aria-hidden="true"]', 0, 1000 * 60, $_WD_OPTION_NoMatch)
+	If @error Then
+		ConsoleWrite("wd_demo.au3: (" & @ScriptLineNumber & ") : The page view is hidden - it is possible that the message about COOCKIE files was not accepted")
```

Comment:
What's a COOCKIE? 😆

Reply:
? 
You may propose any other descripion.

Reply:
The correct spelling is COOKIE